stdlib/public/SDK/Foundation/NSCoder.swift

```diff
@@ -153,6 +160,13 @@ extension NSKeyedUnarchiver {
     try resolveError(error)
     return result
   }
+
+  @nonobjc
+  @available(swift, introduced: 4)
+  @available(OSX 10.11, iOS 9.0, *)
+  public class func unarchiveTopLevelObjectWithData(_ data: Data) throws -> Any? {
```

Comment:
This is the same thing Philippe mentioned. The names already exist as-is; I'm just adding overloads based on the parameter type. If we want to rename them, that'll have to go through API review, no?